package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@azure/logger-js",
+  "version": "0.0.1",
+  "description": "Simple logger for JavaScript projects",
+  "main": "dist/lib/index.ts",
+  "types": "./dist/lib/index.d.ts",
```

Comment:
It doesn't look like you have an `index.ts` file, so you probably want to change `main` and `types` to point to `./dist/lib/logger.js` and `./dist/lib/logger.d.ts`.